src/xcode/ENA/ENA/Source/Services/TicketValidation/Models/ValidationDecorator/ServiceIdentityDocument.swift

```diff
@@ -0,0 +1,11 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct ServiceIdentityDocument: Codable {
```

Comment:
Did you merged the models Artur created into your branch? Because then i think this class (and the file) gets obsolete